journal-of-investigative-dermatology.csl

```diff
@@ -1,227 +1,309 @@
-        </names>
-      </if>
-      <else>
-        <group delimiter=", ">
-          <text variable="container-title" form="short" strip-periods="true" font-style="italic"/>
-          <text variable="collection-title"/>
-        </group>
-      </else>
-    </choose>
-  </macro>
-  <macro name="author">
+  <locale xml:lang="fr">
+    <date form="text" delimiter=" ">
+      <date-part name="day"/>
+      <date-part name="month" form="short" strip-periods="true"/>
+      <date-part name="year"/>
+    </date>
+    <terms>
+      <term name="retrieved">disponible</term>
+      <term name="from">sur</term>
+    </terms>
+  </locale>
+  <locale xml:lang="de">
+    <terms>
+      <term name="retrieved">verfügbar</term>
+      <term name="from">unter</term>
+    </terms>
+  </locale>
+  <macro name="author-short-in-citation">
     <names variable="author">
-      <name delimiter-precedes-last="never" initialize-with="" name-as-sort-order="all"/>
-      <label form="short" strip-periods="false" prefix=" (" suffix=")"/>
-      <et-al font-style="italic"/>
+      <name form="short" and="text"/>
       <substitute>
         <names variable="editor"/>
         <names variable="translator"/>
-        <text macro="title"/>
       </substitute>
     </names>
   </macro>
-  <macro name="author-short">
+  <macro name="author-in-citation">
     <names variable="author">
-      <name form="short" and="text" initialize-with=". "/>
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always"/>
       <substitute>
         <names variable="editor"/>
         <names variable="translator"/>
-        <choose>
-          <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
-            <text variable="title" form="short" font-style="italic"/>
-          </if>
-          <else>
-            <text variable="title" form="short" quotes="true"/>
-          </else>
-        </choose>
       </substitute>
     </names>
   </macro>
+  <macro name="year-in-citation">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="issued-sort">
+    <choose>
+      <if type="article-journal bill book chapter graphic legal_case legislation motion_picture paper-conference report song" match="none">
+        <date variable="issued">
+          <date-part name="year"/>
+          <date-part name="month"/>
+          <date-part name="day"/>
+        </date>
+      </if>
+      <else>
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name delimiter-precedes-last="always" et-al-min="6" et-al-use-first="6" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <label form="long" prefix=", "/>
+      <substitute>
+        <names variable="editor"/>
```

Comment:
`<names variable="translator"/>` since you also have this in the short name displayed in the citation. You almost certainly want those two to line up, since otherwise citations become very hard to link to the bibliography.